tools/jnimarshalmethod-gen/App.cs

```diff
@@ -133,9 +244,192 @@ static void AddRegisterNativeMembers (TypeBuilder dt, ParameterExpression target
 			var lambda  = Expression.Lambda<Action<JniNativeMethodRegistrationArguments>> (body, new[]{ args });
 
 			var rb = dt.DefineMethod ("__RegisterNativeMembers",
-					MethodAttributes.Public | MethodAttributes.Static);
+					System.Reflection.MethodAttributes.Public | System.Reflection.MethodAttributes.Static);
 			rb.SetCustomAttribute (new CustomAttributeBuilder (typeof (JniAddNativeMethodRegistrationAttribute).GetConstructor (Type.EmptyTypes), new object[0]));
 			lambda.CompileToMethod (rb);
 		}
+
+		public static void Error (string message)
+		{
+			Console.Error.WriteLine ($"{Terminal.FgColor (Color.Red, true)}Error: {Name}: {message}{Terminal.ResetColor (true)}");
```

Comment:
Have you tested this on Windows? I'm not sure what using `Terminal.FgColor` will do there.